app/views/my/solutions/widgets/_published.html.haml

```diff
@@ -4,4 +4,4 @@
       %strong Publish this solution.
       Learning from each other’s attempts is a crucial part of Exercism. By making your solution public, other Exercism users will be able to find it and learn from your ideas.
 
-    =link_to "Publish your solution", [:publish, :my, @solution, publish: 1], method: :patch, class: 'pure-button'
+    #=link_to "Publish your solution", [:publish, :my, @solution, publish: 1], method: :patch, class: 'pure-button'
```

Comment:
@kntsoriano has this slipped in by mistake?

Reply:
I needed to do this because rubocop was complaining! It shouldn't matter as the whole file is commented out :)

Reply:
Ah. I missed that. Thanks!